backend/services/members/index.js

```diff
@@ -0,0 +1,54 @@
+const { ApolloServer, gql } = require('apollo-server');
+const { buildFederatedSchema } = require('@apollo/federation');
+const knex = require('../../database');
+const services = require('../index');
+
+const typeDefs = gql`
+extend type Query {
+  me: Member
+}
+
+type Member @key(fields: "stil_id") {
+  stil_id: String!
```

Comment:
alla måste ha ett stil-id? kickar vi ut gamla studenter eller de får lov att ha kvar sina gamla? hur gör vi med admin/demo konton eller om vi behöver ha in icke-studenter? (kanske ett ickeproblem)